### PR TITLE
Show author of PR/issue in link tooltip

### LINKED ISSUE #439: Show author of PR/issue in link tooltip
Currently the link tooltip shows things like the issue title, summary, branch, labels and milestone, but there is no indicator of who created the issue/PR. The task here is to modify the tooltip component to include the author.

image

### PR DESCRIPTION
**Pull request by shinnlok**
#### Summary
Added author to the linked tooltip.

Issue:
Screen Shot 2022-10-23 at 7 48 25 PM

Pull Request:
Screen Shot 2022-10-23 at 7 48 35 PM

#### Ticket Link
Fixes #439

### CODE REVIEW

File: webapp/src/components/link_tooltip/link_tooltip.jsx [hunk 1]
```diff
@@ -97,7 +105,7 @@ export const LinkTooltip = ({href, connected, show, theme}) => {
                             {data.repo}
                         </span>
                         {' on '}
-                        <span>{date}</span>
+                        <span>{date + ' by ' + userName}</span>
```
Comment (hanzei) [inline webapp/src/components/link_tooltip/link_tooltip.jsx:108]: Should the word `by` be shown if the `userName` is empty for what ever reason?

**Review by hanzei — COMMENTED** [review]
Thanks for the PR

File: webapp/src/components/link_tooltip/link_tooltip.jsx (hunk 1 above)
Comment (shinnlok) [reply webapp/src/components/link_tooltip/link_tooltip.jsx]: Updated!

**Review by hanzei — APPROVED** [review]
LGTM :+1:

File: webapp/src/components/link_tooltip/link_tooltip.jsx [hunk 2]
```diff
@@ -97,7 +105,7 @@ export const LinkTooltip = ({href, connected, show, theme}) => {
                             {data.repo}
                         </span>
                         {' on '}
-                        <span>{date}</span>
+                        <span>{date + authorText}</span>
```
Comment (mickmister) [inline webapp/src/components/link_tooltip/link_tooltip.jsx:108]: Can we have the username be a clickable link to the user's GitHub profile?

**Review by mickmister — COMMENTED** [review]
Awesome work @shinnlok! This looks great, I just have one request to use a link to the user's GitHub profile

**Comment by mickmister** [comment]
Hi @shinnlok, just checking in. Are you interested in implementing the piece mentioned in this comment? https://github.com/mattermost/mattermost-plugin-github/pull/597#discussion_r1007135994 Thanks for your hard work on this!

File: webapp/src/components/link_tooltip/link_tooltip.jsx (hunk 2 above)
Comment (mickmister) [reply webapp/src/components/link_tooltip/link_tooltip.jsx:108]: @hanzei Yes this is a blocking comment

**Comment by hanzei** [comment]
Closing due to inactivity

**Pull request closed without merging**